thinkhazard/templates/index.jinja2

```diff
@@ -14,7 +14,9 @@
   </nav>
   <div class="container">
     <header class="text-center">
-      <img src="{{('thinkhazard:static/images/logo-big.png'|static_url)}}" title="Think hazard - Overcome risk" alt="Think hazard - Overcome risk" />
+      <a class="index-brand logo-brand" href="{{'index'|route_url}}">
+        <span class="logo-think">Think</span><span class="logo-hazard">Hazard</span><span class="logo-think">!</span>
```

Comment:
Since we're reusing the same piece of code in both index and report, I'd suggest we factorize it.